torch/distributed/distributed_c10d.py

```diff
@@ -2390,11 +2396,22 @@ def barrier(group=GroupMember.WORLD,
     if _rank_not_in_group(group):
         return
 
+    opts = BarrierOptions()
+    if device_ids is not None:
+        if get_backend(group) !=  Backend.NCCL:
```

Comment:
It looks like the lint failure on this latest version is real:
  {
    path: 'torch/distributed/distributed_c10d.py',
    start_line: 2401,
    end_line: 2401,
    start_column: 33,
    end_column: 33,
    annotation_level: 'failure',
    message: '[E222] multiple spaces after operator'
  }

Reply:
@rohan-varma is the PR imported for merge ?
Thanks.

Reply:
looks like the CI is clear, will reimport and land it now. Thanks for the quick turnaround!